test/FunctionalTests/ProductivityApi/LinqTests.cs

```diff
@@ -12597,7 +12597,8 @@ orderby n.Id
             }
             catch (XunitException ex)
             {
-                Assert.Equal("Assert.Equal() Failure\r\nExpected: 5\r\nActual:   4", ex.Message, ignoreLineEndingDifferences: true);
+                Assert.Contains("Assert.Equal() Failure", ex.Message);
+                Assert.Contains("Expected: 5\r\nActual:   4", ex.Message);
```

Comment:
I might further split this into separate asserts for expected and actual, but I don't feel strongly about it.